### PR TITLE
Auth: Sign Up and Log In 

### PR DESCRIPTION
**Pull request by Tanay-J**
Signup Page:
- Input for First Name and Last Name
- Input for Username
- Input for Password
- Checkbox to accept TnC
- CTA button to Sign Up

Login Page:
- Input for Username
- Input for Password
- Checkbox to stay signed in
- Forgot Password link
- CTA button to Log In

Sign Up Page

image

Log In Page

image

### CODE REVIEW

**Review by anuj-kumary — APPROVED** [review]
Looks good!!

File: css/auth.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.auth-container{
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    max-width: 30rem;
+    background-color: blanchedalmond;
```
Comment (pratik-kumar-sahu) [inline css/auth.css:6]: - use var here for colors

**Review by pratik-kumar-sahu — APPROVED** [review]
- Else looks good. Please go ahead with merging

**Comment by iamharnad** [comment]
great use of classes!

File: css/auth.css (hunk 1 above)
Comment (Tanay-J) [reply css/auth.css:6]: This color is being used only at one place, hence I did not create a variable

**Review by Mahi24-prog — APPROVED** [review]
Looks good, go ahead

**Pull request merged**